### DISCUSSION TITLE
Readability of checkboxes

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [community/community#61999] [Mobile]
Bug: contrast in checkboxes is bad in dark theme. Using latest (beta) app on Android 13.
![Screenshot_20230727-235707.png](https://github.com/community/community/assets/65828559/06326a38-bcb5-4b04-ba5a-92942e66a3a8)

It does show correct (blue background) for many issues and prs. It applies to issues from  https://github.com/CommunityToolkit/Labs-Windows and https://github.com/CommunityToolkit/Windows/issues,
and prs in https://github.com/files-community/Files/pull

Same appearance in Chrome browser.

(Likely related to #54582)

**Comment by aconsuegra** [comment]
Hi @Jay-o-Way, thanks for reaching out with your feedback. As you suggested, this issue is the same as the one reported in https://github.com/orgs/community/discussions/54582, so I'm going to label this discussion as duplicated and close it.
